plugins/spigot/src/main/java/com/managemc/spigot/command/processor/CmdReportsCreate.java

```diff
@@ -0,0 +1,30 @@
+package com.managemc.spigot.command.processor;
+
+import com.managemc.spigot.command.handler.CmdReportsCreateHandler;
+import com.managemc.spigot.command.handler.base.CommandHandlerAsync;
+import com.managemc.spigot.command.processor.tabcompletion.PlayerTabCompleter;
+import com.managemc.spigot.command.util.CommandProcessorAsync;
+import com.managemc.spigot.command.util.CommandUsage;
+import com.managemc.spigot.command.util.ProcessedCommandArguments;
+import com.managemc.spigot.config.SpigotPluginConfig;
+import org.bukkit.command.CommandSender;
+
+public class CmdReportsCreate extends CommandProcessorAsync {
+
+  private final SpigotPluginConfig config;
+
+  public CmdReportsCreate(SpigotPluginConfig config) {
+    super(
+        config.getLogging(),
+        CommandUsage.REPORTS_CREATE,
+        new PlayerTabCompleter(config.getBukkitWrapper())
```

Comment:
Tab completion can be tricky, but this example is one of the simplest and we didn't have to implement anything new in this feature. For more complicated examples, look through the punishment commands.